mount/mount_windows.go

```diff
@@ -55,6 +55,46 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 	return mounter.MountSensitive(source, target, fstype, options, nil /* sensitiveOptions */)
 }
 
+//mountNfs uses powershell command "net use" to connect NFS share
+func mountNfs(source, target string, options []string) error {
+	mountCmd := "powershell /c net use " + source
```

Comment:
do you know `powershell /c net use ` works for container volume mapping?

Reply:
you mean whether it can start container with this? I tried manually and it works for me.

Reply:
I am also wondering whether it can work for smb too.

Reply:
`net use` won't work for smb, that's the reason why we use `SMBGlobalMapping`

Reply:
discussed offline, it can work after creating symlink of local dir to remote network connection.